prometheus/exporter/job_exporter.py

```diff
@@ -83,11 +88,12 @@ def genJobMetrics(logDir, gpuMetrics):
         outputFile.write(containerMemPerc)
 
 def main(argv):
+    logDir = argv[0]
+    timeSleep = int(argv[1])
     while(True):
-        logDir = argv[0]
-        timeSleep = int(argv[1])
+        print("job exporter running")
         # collect GPU metrics
-        gpuMetrics = gpu_exporter.genGpuMetricsFromSmi([logDir])
+        gpuMetrics = gpu_exporter.genGpuMetricsFromSmi(logDir)
         # join with docker stats metrics and docker inspect labels
         genJobMetrics(logDir, gpuMetrics)
```

Comment:
>genJobMetrics [](start = 8, length = 13)

what is the behavior if there is no job running in this machine?

Reply:
will not generate job metrics. and only generate node gpu metrics